apps/core-app/src/Routes.tsx

```diff
@@ -15,13 +22,59 @@ import ProposalDetails from './pages/ProposalDetails';
 import { DaoContainer } from './pages/DaoContainer';
 import { Banner } from '@daohaus/ui';
 import RageQuit from './pages/RageQuit';
+import {
+  HausConnectProvider,
+  HausLayout,
+  useHausConnect,
+} from '@daohaus/daohaus-connect-feature';
+import { useEffect, useLayoutEffect, useState } from 'react';
+
+const HomeContainer = () => {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const { isConnected, address } = useHausConnect();
+  const { profile } = useParams();
+
+  useLayoutEffect(() => {
+    if (isConnected && address && profile) {
+      return;
+    }
+    if (isConnected && !profile) {
+      navigate(`/${address}`);
```

Comment:
Wondering if navigation on connection is what we'd want here.

If we just kept them at the base route, but didn't show the hub data stuff would it be better?
Would enable a user to visit another addresses profile, right?